build-packages.mjs

```diff
@@ -0,0 +1,86 @@
+import fs from "fs";
+import path from "path";
+import del from "del";
+import util from "util";
+import lodash from "lodash";
+const { kebabCase, map } = lodash;
+import childProcess from "child_process";
+import pkg from "./package.json";
+const { peerDependencies } = pkg;
+
+// Convert NodeJS methods to promises in order to use them with `await` statement
+const exec = util.promisify(childProcess.exec);
+const writeFile = util.promisify(fs.writeFile);
+const makeDir = util.promisify(fs.mkdir);
+
+// Read available package sources from the dir
+const modulesArray = [
+  "src/hex.tsx",
+  "src/HexInput.ts",
+  "src/hsl.tsx",
+  "src/hslString.tsx",
+  "src/hsv.tsx",
+  "src/rgb.tsx",
+  "src/rgbString.tsx",
+];
```

Comment:
I figured this was the best/easiest way to target the packages for now, totally could be missing a better option though.

Reply:
I'm okay with this, thanks. The updated building script looks good! But I'm afraid about migrationg to mjs/ESM here.

It supported only since Node 13.2.0 which isn't stable yet.

Reply:
Yeah, I forgot how (not) long ago ESM was actually introduced in a reasonable manner. 

Oh well, no problem reverting. Just a build script anyhow.